src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newDate = date.split(fromFormat[3]);
+  const datePart = {};
+  const formattedDate = [];
+
+  const currentCentury = '20';
+  const previousCentury = '19';
+
+  const CENTURY_INDICATOR = 30;
```

Comment:
You can move current constants outside the function so we can reuse them if needed.
Also, why do they have different naming styles? If the currentCentury and the previousCentury are local variables it makes sense, otherwise lets do them in the same case as CENTURY_INDICATOR.